web/source/kmwuifloat.ts

```diff
@@ -475,7 +479,7 @@ if(!window['keyman']['ui']['name']) {
         else 
         {
           ui.oskButton.style.display = 'block';
-          ui.oskButtonState(osk['isEnabled']());
+          ui.oskButtonState(keymanweb.osk['isEnabled']());
```

Comment:
Do we need to test for `if(keymanweb.osk) ...` here?

Reply:
... that certainly wouldn't hurt.  While everything seems to be functioning fine as-is, the method does appear to be outside of `ui.registerEvents`, so it'd be wise to add a check for robustness.